conformance/utils/echo/pod.go

```diff
@@ -84,8 +86,12 @@ func makeRequest(t *testing.T, exp *http.ExpectedResponse) []string {
 		exp.Request.Method = "GET"
 	}
 
-	if exp.Response.StatusCode == 0 {
-		exp.Response.StatusCode = 200
+	//nolint:staticcheck
+	if exp.Response.StatusCode != 0 {
+		exp.Response.StatusCodes = append(exp.Response.StatusCodes, exp.Response.StatusCode)
+	}
```

Comment:
It took me a while to see there is StatusCode and there is StatusCodes! Probably not a big deal, but would be good to add some comment here on what's changing